src/db-check.hpp

```diff
@@ -0,0 +1,8 @@
+#ifndef OSM2PGSQL_DB_CHECK_HPP
+#define OSM2PGSQL_DB_CHECK_HPP
+
+#include "options.hpp"
+
+void check_db(options_t const &options);
```

Comment:
Can you please document what the function does. The name is rather unspecific.

Reply:
Just pushed a new version that has some documentation.